hummingbot/client/command/history_command.py

```diff
@@ -44,16 +45,18 @@ def history(self,  # type: HummingbotApplication
         self.trade_performance_report()
 
     def balance_snapshot(self,  # type: HummingbotApplication
-                         ) -> Dict[str, Dict[str, Decimal]]:
-        snapshot: Dict[str, Any] = {}
+                         starting_balances: bool = False) -> Dict[str, Dict[str, Decimal]]:
```

Comment:
Would it be simpler for this function to always return the total list? i.e. no `starting_balances` argument required. So it will always return the same number of balances whether it's for starting or current balances.

Reply:
yes absolutely, wasn't sure if that would work with existing logic hence doing it this way – have now changed to the simpler method